src/SFA.DAS.ApplyService.Data/Queries/AppealsQueries.cs

```diff
@@ -25,39 +25,19 @@ private SqlConnection GetConnection()
             return new SqlConnection(_config.SqlConnectionString);
         }
 
-        public async Task<AppealFiles> GetStagedAppealFiles(Guid applicationId)
-        {
-            using (var connection = GetConnection())
-            {
-                var files = (await connection.QueryAsync<AppealFile>(
-                    @"SELECT Id, Filename FROM [AppealUpload] where ApplicationId = @applicationId and AppealId IS NULL ORDER BY CreatedOn ASC",
-                    new
-                    {
-                        applicationId
-                    })).ToList();
-
-                return new AppealFiles
-                {
-                    Files = files
-                };
-            }
-        }
-
-        public async Task<Appeal> GetAppeal(Guid applicationId, Guid oversightReviewId)
+        public async Task<Appeal> GetAppeal(Guid applicationId)
         {
             using (var connection = GetConnection())
             {
                 Appeal result = null;
 
                 await connection.QueryAsync<Appeal, Appeal.AppealUpload, Appeal>(
                     @"SELECT
-                        a.Id, a.Message, a.UserId, a.UserName, a.CreatedOn,
-                        u.Id, u.Filename, u.ContentType
-                        FROM [Appeal] a
-                        LEFT JOIN [AppealUpload] u on u.AppealId = a.Id
-                        JOIN [OversightReview] r ON r.Id = a.OversightReviewId
-                        where a.OversightReviewId = @oversightReviewId
-                        AND r.ApplicationId = @applicationId",
+                        ap.*,
+                        ul.Id, ul.Filename, ul.ContentType
+                        FROM [Appeal] ap
+                        LEFT JOIN [AppealUpload] ul on ul.AppealId = ap.Id
```

Comment:
take out appealId joins imo

Reply:
That's for APR-2640, not this ticket. Have noted it 👍